team.go

```diff
@@ -6,25 +6,13 @@ import (
 	"time"
 )
 
-// Position represents a MyRadio officer position.
-type Position struct {
-	Team        Team   `json:"team"`
-	OfficerID   uint   `json:"officerid"`
-	Name        string `json:"name"`
-	Alias       string `json:"alias"`
-	Ordering    uint   `json:"ordering"`
-	Description string `json:"description"`
-	Status      string `json:"status"`
-	Type        string `json:"type"`
-}
-
 // Officer represents information about an officership inside a Team.
 type Officer struct {
 	User            User `json:"user"`
 	From            time.Time
-	FromRaw         int64    `json:"from"`
-	MemberOfficerID uint     `json:"memberofficerid"`
-	Position        Position `json:"position"`
+	FromRaw         int64           `json:"from"`
+	MemberOfficerID uint            `json:"memberofficerid"`
+	Position        OfficerPosition `json:"position"`
```

Comment:
Is this what `go fmt` comes up with?  I'd have thought it'd justify the tags as well as the types…